Makefile

```diff
@@ -1066,13 +1066,12 @@ ADDON_DOC_LINT_FLAGS=-whitespace/ending_newline,-build/header_guard
 
 lint-cpp: tools/.cpplintstamp
 
-tools/.cpplintstamp: $(LINT_CPP_FILES)
-	@echo "Running C++ linter..."
-	@$(PYTHON) tools/cpplint.py $?
+tools/.cpplintstamp: $(LINT_CPP_FILES) tools/cpplint.py tools/check-imports.py
+	@$(PYTHON) tools/cpplint.py $(LINT_CPP_FILES)
```

Comment:
Is there a way to retain the `$?`?  It provides a nice speed-up in the common case when only some files changed.

Reply:
I have not checked thoroughly but I think the argument in https://github.com/nodejs/node/pull/16581#issuecomment-342757901 applies to the `build/include_what_you_use` rule of cpplint.py.

Reply:
Doesn't really apply, IMO.  The build fails if a deleted file is included.

Reply:
In the JS scenario, other alternatives include "change what named exports a file has" which would cause linter failures in any unchanged file that imported from it - deleting a file was just one example.